bedrock/firefox/templates/firefox/mobile.html

```diff
@@ -302,13 +302,24 @@ <h3>{{ _('Speed') }}</h3>
               </section>
 
               <section class="page-article-section feature-content" id="focus-lightweight" data-section="focus" data-ga-label="firefox focus lightweight">
-                <h3>{{ _('Lightweight') }}</h3>
```

Comment:
When I scroll down the page, the side navigation still lists "Lightweight" as the feature heading, even though the copy here has been updated to centre on privacy. The accompanying image also mentions "Small, yet powerful at less than 5MB.", which now seems out of place.

![screenshot_2018-09-04 mobile browsers for ios and android firefox](https://user-images.githubusercontent.com/400117/45034717-2639e200-b050-11e8-9a02-28a318220bdb.png).

I also don't really see the difference between this new section, and the first "Automatic privacy" section? They both seem to talk about roughly the same thing? :/

![screenshot_2018-09-04 mobile browsers for ios and android firefox 1](https://user-images.githubusercontent.com/400117/45034921-aa8c6500-b050-11e8-8059-101865de2a49.png)

Reply:
Hi Alex - the two are similar, but highlighting different things. My take is that consumers will be interested in both features and brand positioning, which often feel like the same thing from an internal perspective.

"Automatic privacy" refers to features, calling out the automatic blocking of ads and ease of erasing your browser history. These features are available in other products, so to bolster this claim I'm suggesting we reiterate our brand positioning with the third section.

"On your side" refers to the mission that drives the creation of these features, and provides a deeper level of trust concerning the brand behind the product. Akin to talking about improved airbags in a new Toyota model, and then talking about how Toyota is a trusted brand when it comes to safety - similar ideas that enhance one another.

The side navigation should mirror the new subject heading.

The caption should be removed - you're correct that we are moving away from the "small yet powerful" language due to increased Focus Android filesize.